ONSdigital/eq-questionnaire-runner
# Update a schema to demonstrate minimum/maximum supports floats

### What is the context of this PR?
Update the JSON schema validator to accurately support decimal minimum and maximum constraints while disallowing string types. To demonstrate this, I changed a schema in runner to show that floats are allowed.

`test_metadata_routing.json` schema doesn't work and a card in the backlog has been created to address this. The change was to ensure schema validation passes.

### How to review
- Ensure the schema is correct & validates
- Ensure the functional tests (numbers.spec.js) pass

### Checklist
